CONTRIBUTING.md

```diff
@@ -96,5 +87,5 @@ and that a record of the contribution (including all personal information I
 submit with it, including my sign-off) is maintained indefinitely and may be
 redistributed consistent with this project or license(s) involved.
 
-[CoC]: CODE_OF_CONDUCT.md
-[DCO]: https://developercertificate.org/
+[coc]: CODE_OF_CONDUCT.md
+[dco]: https://developercertificate.org/
```

Comment:
why those changes in this file?

Reply:
My editor had gone crazy until I fixed `.vscode/settings.json`. No idea why it chose to convert the upper case to lower 😆 
I'm still finalizing this PR, will request reviews once I'm ready.